### PR TITLE
Add FAQ page

### LINKED ISSUE #300: Unity personal activation fails on OneDev CI
OneDev secrets breaks encoding of ulf license file. This is not a bug in unity or game-ci images but I think it is worth to include solution in troubleshooting docs.

I can create a pull request with description how to workaround this issue. It is best to store ulf file content in secret encoded as a base64 string and decode it on runner that needs it.

`bash -c "/bin/echo $UNITY_LICENSE > /opt/project/Unity_v2022.x.base64 && base64 -d  /opt/project/Unity_v2022.x.base64 >  /opt/project/Unity_v2022.x.ulf"`

Error:
`Error: Code 400 while processing request (status: Cannot save ULF license file: Validation failed. The digital signature is invalid.)`

OneDev:
[https://github.com/theonedev/onedev](url)

Edit: I should have marked this as documentation, not a bug.

### PR DESCRIPTION
**Pull request by GabLeRoux**
#### Changes

- Add FAQ page
- Add some links to FAQ
- Add a redirect so we can call `/faq`

#### Checklist

- [x] Read the contribution [guide](../blob/main/CONTRIBUTING.md) and accept the
      [code](../blob/main/CODE_OF_CONDUCT.md) of conduct
- [x] Readme (updated or not needed)
- [x] Tests (added, updated or not needed)

Closes #300

### CODE REVIEW

**Comment by webbertakken** [comment]
Could we maybe also add a subtitle "Frequently asked questions"? 

Younger non-native English speakers might not know what FAQ is at first. I believe it would help make it slightly more accessible.

File: docs/10-faq/index.mdx [hunk 1]
```diff
@@ -0,0 +1,262 @@
+---
+title: FAQ
+---
```
Comment (webbertakken) [inline docs/10-faq/index.mdx:3]: Everywhere else we do this using a [\_category_.yaml](https://github.com/game-ci/documentation/blob/main/docs/09-troubleshooting/_category_.yaml) file, which gives more flexibility.

**Review by webbertakken — APPROVED** [review]

**Comment by GabLeRoux** [comment]
> [ERROR] Error: Docusaurus found broken links!
😱 fixing that right away, this check is sooooo nice <3

File: docs/09-troubleshooting/common-issues.mdx [hunk 2]
```diff
@@ -2,8 +2,93 @@
 
 Below you'll find a collection of answers to commonly reported problems.
 
+## Unity Activation
```
Comment (GabLeRoux) [inline docs/09-troubleshooting/common-issues.mdx:5]: The diff here might be harder to follow, but I moved issues related to activation on top as they are the most common ones so far. Then I added some new sections. 👍

File: docs/09-troubleshooting/common-issues.mdx [hunk 3]
```diff
@@ -2,8 +2,93 @@
...
+
+If you are having troubles with activation, you are not alone, this is a common problem and it is
+not the easiest thing to solve. Here are some tips:
+
+- Try removing special characters from your password, sometimes they cause problems with command
+  lines. _This is not a security advice_
+- Your username should be an email
+- Confirm you are able to connect on Unity's website using your credentials
+
+:::caution Known issue with gitlab-ci activation and empty license file
```
Comment (GabLeRoux) [inline docs/09-troubleshooting/common-issues.mdx:15]: We will want to remove that once this is solved on the gitlab side, but this should help us a bit by pointing people in the right direction until this gets fixed. There has been a few people asking for this already on discord.

File: docs/05-gitlab/02-activation.mdx [hunk 4]
```diff
@@ -3,6 +3,26 @@
 There are a few methods available, if you're using gitlab-ci, the easiest method in the current
 documentation is using gitlab-ci.
 
+:::caution Known issue with gitlab-ci activation and empty license file
```
Comment (GabLeRoux) [inline docs/05-gitlab/02-activation.mdx:6]: Copy paste: We will want to remove that once this is solved on the gitlab side, but this should help us a bit by pointing people in the right direction until this gets fixed. There has been a few people asking for this already on discord.

File: docs/09-troubleshooting/common-issues.mdx [hunk 5]
```diff
@@ -2,8 +2,93 @@
...
+
 ## My workflow isn't working because
 
+### I get `manifest for unityci/editor:2021.3.0f1-android-0 not found: manifest unknown: manifest unknown`
+
+This means the docker image trying to download from docker hub is not found.
+
+### Solution
+
+In this specific situation, the pipeline is using an older
```
Comment (GabLeRoux) [inline docs/09-troubleshooting/common-issues.mdx:90]: Oups, this is not complete, fixing that right away.

Comment (GabLeRoux) [reply docs/09-troubleshooting/common-issues.mdx]: Fixed

File: docs/09-troubleshooting/common-issues.mdx [hunk 6]
```diff
@@ -30,6 +152,42 @@ Make sure your branch is clean and all files are indeed present:
 A good way to verify this, is to (locally) clone the Unity project in a new folder and run the build
 from there.
 
+### Branch is dirty
+
+#### Error
+
+```plaintext
+Branch is dirty. Refusing to base semantic version on uncommitted changes
```
Comment (GabLeRoux) [inline docs/09-troubleshooting/common-issues.mdx:160]: Is this also something that can happen when LFS failed or something? Thinking of this, it might be interesting to mention a few things concerning git LFS, but that can definitely be in a different pr :)

**Comment by GabLeRoux** [comment]
I consider this ready to merge 🚀

File: docs/10-faq/index.mdx (hunk 1 above)
Comment (GabLeRoux) [reply docs/10-faq/index.mdx]: Fixed, I updated the title directly so we don't have weird title levels. `Frequently Asked Questions (FAQ)`.

File: docs/09-troubleshooting/common-issues.mdx (hunk 6 above)
Comment (webbertakken) [reply docs/09-troubleshooting/common-issues.mdx:160]: Yes, it's possible if you don't check out your files then your branch is dirty
